app/scripts/modules/amazon/loadBalancer/details/loadBalancerDetail.controller.js

```diff
@@ -48,7 +46,7 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.details.controller',
             }
 
             $scope.loadBalancer.elb.securityGroups.forEach(function (securityGroupId) {
-              var match = securityGroupReader.getApplicationSecurityGroup(app, loadBalancer.accountId, loadBalancer.region, securityGroupId);
+              var match = SecurityGroupReaderService.getApplicationSecurityGroup(app, loadBalancer.accountId, loadBalancer.region, securityGroupId);
```

Comment:
1. The service should just be named `SecurityGroupReader` or `SecurityGroupReadService`, not `SecurityGroupReaderService`

2. While it works right now to make this method static, doing so prevents us from being able to use injected fields in the future, so I'd much rather leave it as a non-static method.